package.json

```diff
@@ -12,10 +12,13 @@
     "postinstall": "cd packages/react-instantsearch && npm install",
     "doctoc": "doctoc --maxlevel 1 README.md CONTRIBUTING.md",
     "test": "(cd packages/react-instantsearch && npm test) && npm run lint && NODE_ENV=production DOCS_MOUNT_POINT=/instantsearch.js/react/ DOCS_DIST=docs/react/ npm run docs:build && NODE_ENV=production npm run build",
-    "dev": "cd packages/react-instantsearch && npm run dev"
+    "dev": "cd packages/react-instantsearch && npm run dev",
+    "storybook": "start-storybook -p 6006",
```

Comment:
I use storybook mainly because of issues in the doc side conf and I wanted to move on. Also it guaranteed me that it was a complete isolated env and it's quite fast when you want to do a lot of little experiments/variations. 

But I believe once those issues will be solved, we will have the same things with our widgets demo pages. I push it because I wanted to have a look at the default styling but It was not intended to last. 

As for storybook, it's still under development so they add some nice stuff regularly like snapshoting for free this week. But for now I don't believe it could feet nicely within our doc. For example Airbnb add it as a complete external website (http://airbnb.io/react-dates)